strawberry/sanic/views.py

```diff
@@ -0,0 +1,79 @@
+import json
+
+from sanic.exceptions import abort
+from sanic.response import HTTPResponse, html
+from sanic.views import HTTPMethodView
+from strawberry.http import GraphQLHTTPResponse, process_result
+from strawberry.types import ExecutionResult
+
+from ..schema import BaseSchema
+from .graphiql import render_graphiql_page
+
+
+class GraphQLView(HTTPMethodView):
```

Comment:
As this is a class where users are expected to create subclasses from, we should provide some docstring on what each function does.

Reply:
Agreed, added a doc-string

Reply:
On the methods too :)